webapp/src/main/webapp/iaf/gui/js/directives.js

```diff
@@ -367,6 +391,77 @@ angular.module('iaf.beheerconsole')
 	};
 }])
 
+.directive('ngMermaid', ['$timeout', function ($timeout) {
```

Comment:
Ik denk dat het beter/handiger is dit als losse file te includen (met commentaar bovenaan waar hij vandaan komt) volgens mij https://github.com/MSakamaki/ng-mermaid?

Reply:
Zou je een `js/plugins/mermaid/ng-mermaid.js` kunnen maken, en deze file via de lazyload kunnen includen?

Reply:
Niet zomaar 'resolven' .. :(